sematic/ui/packages/main/src/sockets.ts

```diff
@@ -1,5 +1,14 @@
 import io from "socket.io-client";
 
 export const graphSocket = io("/graph");
+export const jobSocket = io("/job");
+export const pipelineSocket = io("/pipeline");
 
-export const pipelineSocket = io("/pipeline");
\ No newline at end of file
+// TODO: remove once job update socket is hooked to something
```

Comment:
As far as I can see, this is the only place where the update is listened for. Is that correct? Do you have plans to expand this functionality?

Reply:
The UI will want to refresh itself when jobs are updated. That will come as separate work done by Chance. This just tests that the updates are in fact getting sent to the front end and give Chance the place to start.